pkgs/build-support/ocaml/oasis.nix

```diff
@@ -1,14 +1,14 @@
 { lib, stdenv, ocaml_oasis, ocaml, findlib, ocamlbuild }:
 
 { pname, version, nativeBuildInputs ? [], meta ? { platforms = ocaml.meta.platforms or []; },
-  minimumOCamlVersion ? null,
+  minimalOCamlVersion ? null,
```

Comment:
Did you change this one on purpose? If you really want to clean up this `buildOasis…` stuff, then you should also fix how this attribute is handled (i.e., not given to `mkDerivation`).

Reply:
I'm not sure I properly understand what you mean. I have replaced the references of `minimumOCamlVersion` with `minimalOCamlVersion`. What do you suggest to be done exactly?

Reply:
That you don’t touch this file.

Reply:
Dropped changes to oasis file.

Reply:
Then please also drop the changes to ocaml-sat-solvers and tcslib.

Reply:
Dropped changes to ocaml-sat-solvers and tcslib.